jsx/StaticDataTable.js

```diff
@@ -193,14 +267,8 @@ StaticDataTable = React.createClass({
                     data = this.props.Data[index[i].RowIdx][j];
                 }
 
-                // Increase counter, if filter value is found to be a substring
-                // of one of the column values
-                var filterData = this.props.Filter[this.props.Headers[j]];
-                if (filterData !== null &&
-                    data !== null &&
-                    data.indexOf(filterData) > -1
-                ) {
-                    filterMatchCount++;
+                if (this.hasFilterKeyword(this.props.Headers[j], data)) {
```

Comment:
I might be missing something but won't this count the same row multiple times if multiple columns in the same row match the filter?